src/app/app.ts

```diff
@@ -183,6 +184,35 @@ export function view(state: IState, actions: any) {
             actions.graph.deleteNode,
           )
         : Empty(),
-    ],
-  )
-}
+
+        html.div(
+          {
+            herf: "#",
+            id: "drag",
+            style: {
+              position: 'absolute',
+              bottom: 0,
+              right: 0,
+              height: '200px',
+              width: '200px',
+              background: 'White',
+            },
+          }
+
+            var dragFile = document.getElementById('drag')
```

Comment:
Lines 202 to 211 must be placed inside an event such as oncreate or onclick for this to work.